qiskit/transpiler/passes/utils/check_map.py

```diff
@@ -43,18 +43,22 @@ def run(self, dag):
         """
         self.property_set["is_swap_mapped"] = True
 
-        if self.coupling_map is None:
+        if self.coupling_map is None or len(self.coupling_map.graph) == 0:
             return
 
+        self.coupling_map.compute_distance_matrix()
+
         qubit_indices = {bit: index for index, bit in enumerate(dag.qubits)}
+        # Use dist matrix directly to avoid validation overhead
+        dist_matrix = self.coupling_map._dist_matrix
```

Comment:
Same as above, can these be combined into a single call like `dist_matrix = self.coupling_map.distance_matrix`?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7036/commits/657d605285fa19576d1aeb865dd57b8e9a99f6bc